_maps/map_files/Blueshift/Blueshift.dmm

```diff
@@ -32762,9 +32764,6 @@
 	pixel_x = 6;
 	pixel_y = 16
 	},
-/obj/item/reagent_containers/cup/beaker/vial,
```

Comment:
apparently this path doesn't exist, so strongdmm nuked it

Reply:
I wonder what it was changed to.